Tests/_CollectionsTestSupport/AssertionContexts/Assertions.swift

```diff
@@ -300,6 +300,22 @@ public func expectEqualElements<S1: Sequence, S2: Sequence>(
     message, trapping: trapping, file: file, line: line)
 }
 
+public func expectNotEqualElements<S1: Sequence, S2: Sequence>(
```

Comment:
Huh, perhaps we should have some sort of DSL for these expectations. But this is fine!